src/explore-education-statistics-common/src/modules/table-tool/components/utils/downloadTableOdsFile.ts

```diff
@@ -108,6 +109,7 @@ export function appendFootnotes(
   sheet: WorkSheet,
   footnotes: FullTableMeta['footnotes'],
 ): WorkSheet {
+  let linkHref: string;
```

Comment:
- Minor, but would suggest adding a line beneath to space things out
- Would add a comment explaining that this is a bit of a workaround to get around sanitize-html's API